index.html

```diff
@@ -347,5 +347,104 @@ <h2>Examples</h2>
       </pre>
     </div>
   </section>
+  <section>
+    <h2>Transferable MediaStreamTrack</h2>
+    <div>
+      <p>{{MediaStreamTrack}} are <a data-cite="!HTML/#transferable-objects">transferable objects</a>.
+      This allows manipulating real-time media outside the context it was requested or created, for instance in workers or third-party iframes.</p>
+      <p>To preserve the existing privacy and security infrastructure, in particular for capture tracks,
+      a transferred track remains tightly bound to the original track.
+      In particular, if the original track gets muted, unmuted or ended, the transferred track will automatically be muted, unmuted or ended.
+      It is in particular important to ensure that media will stop flowing as soon as the context of the original track goes away.</p>
+    </div>
+    <div>
+      <p>The WebIDL changes are the following:
+      <pre class="idl"
+>[Exposed=(Window,Worker,AudioWorklet), Transferable]
+partial interface MediaStreamTrack {
+// Remove partial once we figure out how to do it without creating respec warnings.
+// No change to MediaStreamTrack exposed API.
+};</pre>
+    </div>
+    <div>
+      <p>At creation of a {{MediaStreamTrack}} object, called <var>track</var>, run the following steps:</p>
+      <ol>
+        <li><p>Initialize <var>track</var>.`[[IsDetached]]` to <code>false</code>.</p></li>
+      </ol>
+    </div>
+    <div>
+      <p>{{MediaStreamTrack}} <a data-cite="!HTML/#transfer-steps">transfer steps</a>, given <var>value</var> and <var>dataHolder</var>, are:</p>
+      <ol>
+        <li><p>If <var>value</var>.`[[IsDetached]]` is <code>true</code>, throw a "DataCloneError" DOMException.</p></li>
+        <li><p>Set <var>dataHolder</var>.`[[id]]` to <var>value</var>.{{MediaStreamTrack/id}}.</p></li>
+        <li><p>Set <var>dataHolder</var>.`[[kind]]` to <var>value</var>.{{MediaStreamTrack/kind}}.</p></li>
+        <li><p>Set <var>dataHolder</var>.`[[label]]` to <var>value</var>.{{MediaStreamTrack/label}}.</p></li>
+        <li><p>Set <var>dataHolder</var>.`[[readyState]]` to <var>value</var>.{{MediaStreamTrack/readyState}}.</p></li>
+        <li><p>Set <var>dataHolder</var>.`[[enabled]]` to <var>value</var>.{{MediaStreamTrack/enabled}}.</p></li>
+        <li><p>Set <var>dataHolder</var>.`[[muted]]` to <var>value</var>.{{MediaStreamTrack/muted}}.</p></li>
+        <li><p>Set <var>dataHolder</var>.`[[source]]` to <var>value</var> underlying source.</p></li>
+        <li><p>Set <var>dataHolder</var>.`[[constraints]]` to <var>value</var> active constraints.</p></li>
+        <li><p>Let <var>port1</var> be a new {{MessagePort}} in <a href="https://tc39.es/ecma262/#current-realm">the current Realm</a>.</p></li>
+        <li><p>Let <var>port2</var> be a new {{MessagePort}} in <a href="https://tc39.es/ecma262/#current-realm">the current Realm</a>.</p></li>
```

Comment:
I'm not convinced we need these ports. They seem to be used solely to convey signals from the user agent — `track.stop()` on the detached track won't work — which I think may be better described as requirements on the user agent, if they aren't already, without going through the original track. This also avoids complexity if a transferred track is transferred around further. 

E.g. I'd expect user agents to mute/unmute or end the _source_, not individual tracks. To the extent this is not the case, I wouldn't begin to assume how this might work in a particular user agent, nor would I try to prescribe it.

We should probably audit our [permission indicator logic](https://w3c.github.io/mediacapture-main/getusermedia.html#mediadevices) after this though.